src/components/settings-menu/SettingsSection.js

```diff
@@ -379,6 +380,14 @@ export default function SettingsSection({
             onPress={onPressDev}
             testID="developer-section"
           />
+          {IS_DEV && (
```

Comment:
This needs to be dev or testflight, no?

Reply:
dang you're right i missed testflight here